Birthday-2024-Project/export_presets.cfg

```diff
@@ -8,7 +8,7 @@ custom_features=""
 export_filter="all_resources"
 include_filter=""
 exclude_filter=""
-export_path="../../Builds/Windows/FaunasOrganizedNeatlyLevelEditor.exe"
```

Comment:
was it an intended change?

Reply:
With this and the other problem, I did not change anything other than adding files and assigning the screenshots to the levels, I don't know why these other things have changed

Reply:
Both problems should be fixed now